packages/@aws-cdk/aws-appmesh/test/test.gateway-route.ts

```diff
@@ -133,6 +133,39 @@ export = {
       /Prefix Path must start with \'\/\', got: wrong/);
       test.done();
     },
+
+    'with shared service mesh': {
+      'Mesh Owner is the AWS account ID of the account that shared the mesh with your account'(test:Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
```

Comment:
Let's explicitly set the account of the Stack to be different than the account of the Mesh, to make sure it's clear what's the purpose of this test.

Reply:
Yes that makes sense.